packages/flutter/lib/src/widgets/ticker_provider.dart

```diff
@@ -94,7 +95,10 @@ abstract class SingleTickerProviderStateMixin<T extends StatefulWidget> extends
         'mixing in a SingleTickerProviderStateMixin, use a regular TickerProviderStateMixin.'
       );
     }());
-    _ticker = new Ticker(onTick, debugLabel: 'created by $this');
+    final ValueListenable<AccessibilityFeatures> accessibilityFeatures = SemanticsBinding.instance.accessibilityFeatures;
```

Comment:
fwiw, as a general rule we try to get things like this from the tree (e.g. via a MediaQuery) rather than from the binding directly

Reply:
That doesn't work here, the tickers are created before we can use context to get the media query data, and I found (confusingly) that animations would then start before didChangeDependencies could set the correct behavior.

Reply:
makes sense, they'll get created from initState.

But that shouldn't matter, right? The animation controller will only have done the first (zero) frame by the time you get didChangeDependencies called, so there won't be any difference if you set the state then or at creation time.